distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/carbon.xml.j2

```diff
@@ -530,6 +530,13 @@
                 <!--Pattern></Pattern-->
             </Patterns>
         </XSSPreventionConfig>
+
+        {%if security.sts.signature_algorithm is defined %}
+        <STSSignatureAlgorithm>{{security.sts.signature_algorithm}}</STSSignatureAlgorithm>
```

Comment:
Do we need 'security' here? can we have just sts?

Reply:
removed security